<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Stock_Update_Part</class>
 <widget class="QDialog" name="Stock_Update_Part">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update Part</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/images.qrc">
    <normaloff>:/emoticons/service.png</normaloff>:/emoticons/service.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="2" alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QDoubleSpinBox" name="double_Spin_Cost_Price">
     <property name="maximum">
      <double>999.990000000000009</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="4" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="6">
    <widget class="QPlainTextEdit" name="txt_Part_Description">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="3" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbl_Sell_Price">
     <property name="text">
      <string>     Set Interest Rate %</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbl_Brand">
     <property name="text">
      <string>Brand</string>
     </property>
    </widget>
   </item>
   <item row="5" column="5">
    <widget class="QLabel" name="lbl_Quantity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>     In Stock</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" alignment="Qt::AlignLeft">
    <widget class="QDoubleSpinBox" name="double_Spin_Interrest_Rate">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <double>999.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="5" column="6">
    <widget class="QSpinBox" name="spin_Quantity">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="1" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="lbl_Cost_Price">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Cost Price</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignRight|Qt::AlignTop">
    <widget class="QLabel" name="lbl_Description">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbl_Part_id">
     <property name="text">
      <string>Part ID</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="line_Final_Sell_Price">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="lbl_Final_Price">
     <property name="text">
      <string>Final Price</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" alignment="Qt::AlignRight">
    <widget class="QLabel" name="lbl_Part_Name">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QLineEdit" name="line_Name">
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>50</number>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="line_Brand"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="line_Part_id">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>line_Name</tabstop>
  <tabstop>line_Brand</tabstop>
  <tabstop>txt_Part_Description</tabstop>
  <tabstop>double_Spin_Cost_Price</tabstop>
  <tabstop>double_Spin_Interrest_Rate</tabstop>
  <tabstop>spin_Quantity</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>line_Final_Sell_Price</tabstop>
  <tabstop>line_Part_id</tabstop>
 </tabstops>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
